langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -75,6 +80,14 @@ type XmlType xml;
 # Returns an iterator over the xml items of an xml sequence.
 #
 # # Each item is represented by an xml singleton.
+# 
+# ```ballerina
+# xml books = xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`;
+# object {
+#   public isolated function next() returns record {|xml value;|}?;
+# } iterator = books.iterator();
+# record {|xml value;|}? next = iterator.next();
```

Comment:
```suggestion
# iterator.next() ⇒ {"value":`<book><title>Ballerina</title></book>`}
```

Reply:
When ever possible let's add the outputs

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/f0dac9d10c92de79bf074968b42be066e0b45d1c